helmfile.d/snippets/derived.gotmpl

```diff
@@ -31,6 +34,19 @@ environments:
     values:
       - _derived: # < introduced to hold compound logic in meaningful prop names for easier consumption
           untrustedCA: {{ or (eq $issuer "custom-ca") (and (eq $issuer "letsencrypt") (eq ($cm | get "stage" "") "staging")) }}
+          {{- if $c.keycloak.enabled }}
+          openidName: {{ $k | get "idp.alias" "otomi-idp" }}
+          openidBaseUrl: {{ printf "https://keycloak.%s/realms/otomi" $domainSuffix }}
+          openIdWellKnownUrl: {{ printf "https://keycloak.%s/realms/otomi/.well-known/openid-configuration" $domainSuffix }}
+          openIdClientId: {{ $k.idp.clientID  }}
+          openIdClientSecret: {{ $k.idp.clientSecret }}
+          {{- else }}
+          openidName: "otomi"
+          openidBaseUrl: {{ $o | get "issuer" nil }}
+          openIdWellKnownUrl: {{ printf "%s/.well-known/openid-configuration" ($o | get "issuer" nil) }}
+          openIdClientId: {{ $o | get "clientID" nil }}
+          openIdClientSecret: {{ $o | get "clientSecret" nil }}
+          {{- end }}
```

Comment:
I strongly disagree to (ab)use this mechanism and did not want to see it introduced in the first place.

Let me present an alternative that avoids adding new props. I will add a commit.